### DISCUSSION TITLE
访问(下载)某些文件的时候会出现403被拒绝

### DISCUSSION THREAD

**Discussion by tanmantang** [spencerwooo/onedrive-vercel-index#597] [Q&A] [answered]
如下图，有人遇到过吗，不知道为啥，其他的都是正常的

例子：https://pan.tanmantang.com/api/raw/?path=/%E4%B8%80%E4%BA%9B%E5%A5%BD%E7%94%A8%E7%9A%84%E8%BD%AF%E4%BB%B6/navicat12.0.27_premium_cs_x64.zip

![image](https://user-images.githubusercontent.com/42775204/163205337-38fc3568-ac07-4e4a-ad79-6fe27867dcf1.png)

**Comment by opoet** [comment, marked as answer]
#525

**Reply by tanmantang** [reply]
> #525

看了下连接，只有非个人版的解决方案，有那种个人版的解决方案不。感谢大佬了

**Reply by spencerwooo** [reply]
> 只有非个人版的解决方案，有那种个人版的解决方案不。

其实可以考虑使用 E5 开发者版本，如果实在无法解决这个问题的话。（我合理怀疑是文件被 OneDrive 风控了）
